jenkins/spark-nightly-build.sh

```diff
@@ -115,11 +115,13 @@ distWithReducedPom "install"
 if [[ $SKIP_DEPLOY != 'true' ]]; then
     distWithReducedPom "deploy"
 
+TOOL_PL=${TOOL_PL:-"tools"}
```

Comment:
nit indentation, although I think I would rather see this at the top of the file to make it more obvious to user they could override